net.sourceforge.tomatoes.IHaveNoTomatoes.json

```
@@ -0,0 +1,107 @@
+{
+  "app-id": "net.sourceforge.tomatoes.IHaveNoTomatoes",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "tomatoes",
+  "cleanup": [
+    "/include",
+    "*.a",
+    "*.la",
+    "/bin/libmikmod-config",
+    "/bin/sdl-config",
+    "/lib/pkgconfig",
+    "/share/aclocal",
+    "/share/info",
+    "/share/man"
+  ],
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--persist=.tomatoes"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.json",
+    "shared-modules/SDL/sdl12-compat.json",
+    "shared-modules/SDL/SDL_image-1.2.12.json",
+    {
+      "name": "libmikmod-2",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/mikmod/files/libmikmod/3.1.21.1/libmikmod-3.1.21.1.tar.gz",
+          "sha256": "1a88922b5a67c308899c1d991ddfb62b5d851f8a94175612b33856854938b2fb",
+          "x-checker-data": {
+              "type": "html",
+              "url": "https://sourceforge.net/projects/mikmod/rss",
+              "pattern": "<link>(https://sourceforge.net/.+/libmikmod-3\.1\.([\\d\\.]+\\d).tar.gz)/download</link>"
+          }
+        }
+      ]
+    },
+    "shared-modules/SDL/SDL_mixer-1.2.12.json",
+    {
+      "name": "tomatoes-data",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/share/tomatoes/",
+        "cp -r config.cfg music tomatoes.mpk /app/share/tomatoes/"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/tomatoes/tomatoes-linux-1.5.tar.bz2",
+          "sha256": "69a3af1994cb58409be9b469e1aa4bdb91c405d99071c7431526640ec53d1300"
+        }
+      ]
+    },
+    {
+      "name": "tomatoes",
+      "buildsystem": "simple",
+      "build-commands": [
+        "make",
+        "install -Dm755 tomatoes -t /app/bin/",
+        "install -Dm644 net.sourceforge.tomatoes.IHaveNoTomatoes.appdata.xml -t /app/share/metainfo/",
+        "install -Dm644 net.sourceforge.tomatoes.IHaveNoTomatoes.desktop -t /app/share/applications/",
+        "install -Dm644 icon.png /app/share/icons/hicolor/64x64 -t /apps/net.sourceforge.tomatoes.IHaveNoTomatoes.png"
```

```suggestion
        "install -Dm644 icon.png /app/share/icons/hicolor/64x64/net.sourceforge.tomatoes.IHaveNoTomatoes.png"
```

not sure what happened but this is the right line.

---

\*chuckle* I should have done a quick build run locally but I just took it for granted that you wouldn't make any mistakes. Could have saved us a buildbot failure.

---

no biggie.